src/coreComponents/mesh/generators/VTKMeshGenerator.cpp

```diff
@@ -464,34 +469,153 @@ std::vector< T > collectUniqueValues( std::vector< T > const & data )
 
   return allData;
 }
+/**
+ * @brief Identify the type of the polyhedron
+ *
+ * @param cell The VTK cell, type polyhedron (42)
+ * @return int The type of polyhedron:
+ *              =0 represent a general polyhedron
+ *              >0 correspond to the equivalent standard vtk cell (10, 12, 13, 14, 15, 16)
+ *             N<0 identify a prism with (-N) sides
+ */
+int polyhedronType( vtkCell * const cell )
+{
+  localIndex numberOfFaces = cell->GetNumberOfFaces();
+  localIndex numberOfQuads = 0;
+  localIndex numberOfTriangles = 0;
+  set< localIndex > ListQuadsPoints;
+  set< localIndex > ListNoQuadsPoints;
+
+  if( cell->GetCellType() != VTK_POLYHEDRON )
+    GEOSX_ERROR( "Input for polyhedronType() must be a VTK_POLYHEDRON (42)." );
```

Comment:
```suggestion
  if ( cell->GetCellType() != VTK_POLYHEDRON )
  {
    GEOSX_ERROR( "Input for polyhedronType() must be a VTK_POLYHEDRON (42)." );
  }
```

Always use curly braces (GEOSX [code style](https://geosx-geosx.readthedocs-hosted.com/en/latest/docs/sphinx/developerGuide/Contributing/CodeStyle.html#code-format))